app/services/requests/user/user-service.js

```diff
@@ -19,4 +20,21 @@ export default class RequestsUserUserService extends Service {
       method: 'GET',
     });
   }
+
+  async createUser(user) {
+    const headers = new Headers();
+    headers.append('Content-Type', 'application/json'); // Set the request content type to JSON
```

Comment:
This is another candidate for a new middleware. Let's keep an eye on it, and if you see the pattern how it appears you can add a middleware later on :ok_hand: